### DISCUSSION TITLE
好不習慣中央的天氣

### DISCUSSION THREAD

**Discussion by lai1008** [1101-GS4719-C/Python-Community-GS4719-C#2217] [General]
不知道有沒有南部上來的同學跟我一樣很不適應北部還沒到真正的冬天，天氣就轉冷，而且風還超大好不習慣。

**Comment by Chattenoire** [comment]
來這裡是來修行的。
天氣爆烈、校園行政/領導者沒有遠見、食物難吃、畢業系統食古不化，一切只能靜觀，修身養性。

阿彌陀佛，耶穌保佑，阿拉領路，耶和華祝福，玉皇大帝在上，濕婆平安🙏

**Comment by 113607514** [comment]
雖然我是桃園人，但中央的天氣真的太惡劣了，原本今天要去上早八的課，但天氣真的太惡劣好濕好冷，就讓我打消了上課的念頭。

**Comment by christaaah** [comment]
同南部人，但是已經習慣這種天氣了，我只能建議暖暖包多準備幾個，加購圍巾大衣等南部根本不會用到的東西，最後睡覺的時候穿襪子，馬上就會暖很多，超級有效。

**Comment by 112121py** [comment]
中部人簽，以前在家裡常常不會有颱風災情很嚴重的情況，大佛都會幫我們擋（respect），一上來就是各種大小雨還有不知道怎麼吹的風，每天走去上課像在玩生存遊戲，冬天褲子還會濕爛，爛透了但我也習慣了qq

**Comment by forestdew507** [comment]
雨傘可以買一把大傘，我的小折疊傘在中央被吹壞兩支了QQ

**Comment by gigitilto** [comment]
我很喜歡這裡的天氣，雖然常常颳風下雨，腳踏車很容易生鏽，鞋子也很容易濕透，但我很喜歡雨水冰冰涼涼的感覺，雨天的景致亦別有一番風味，平息我躁動的心情。

**Comment by sRain0113** [comment]
而且時不時會下噴霧狀的毛毛雨，風一大就算撐傘，雨也會直接滲透到衣服裡面，很常因為這樣著涼。非常敬佩本來就住在北部的人，可以在這種天氣下生存。

**Comment by poyen0806** [comment]
日本人簽，日本那裡雖然溫度較低，但是濕度卻沒台灣那麼誇張，所以感受起來舒適很多。來台灣後我不能離開除濕機，我會直接發霉。

**Comment by yue083139** [comment]
中部人簽，以前在高中的時候甚至有一年以上面見過雨，來到中央後颳風下雨走在路上雨傘只能擋上半身真的好討厭，然後被淋濕了之後加上低溫直接去世，而且一直下雨每次晾衣服都很久才會乾，感覺唯一的好處大概只有上大學後我的鼻子過敏好了不少這一點而已。

**Comment by Tchang323** [comment]
中央的天氣陰晴不定~~中央的天氣真的說變就變
不過對於台北人來說，中央已經算是比較不會下雨? 我感覺在台北真的比這裡潮濕很多，不確定是不是天氣的原因，但每次回家都很容易長痘痘。

**Comment by ShuliANN** [comment]
下雨天的中央真的讓人不愉快，不想出門上課。不過聽別的同學在台北更常下雨，就覺得我們已經算好了，而且一旦是晴天，整個中央公園都很美，其實也還能接受這樣的天氣（我也蠻喜歡冷天的！

**Comment by yuan3x2** [comment]
生為土生土長的桃園人，20年來從來沒有習慣過，每到冬天又濕又冷都讓我很頭痛，起床更是痛苦，衣服也都曬不乾，出門也超不方便！！！只能說中壢爛天氣。

**Comment by Mminniehuang** [comment]
莫名其妙的天氣啊😠來到這裡的第一年真的驚呆了
我是從南部來的，冬天的時候甚至寒流來也都才18度左右，那時候就已經有一股要冷死的感覺，但自從來到這裡後，第一次感受到冬天有15度以下，現在冬天回去南部都還可以穿短袖...幸好今年的冬天有比較溫柔一點，不然我真的不知道怎麼活下去。

**Comment by Elaine4488** [comment]
之前住基隆，對於冬天每天下雨的天氣已經見怪不怪了。相比之下，我覺得中央已經沒那麼常下雨了，但一樣很潮濕，建議大家都要有一台除濕機！

**Comment by qiu0109** [comment]
身為桃園人我也不太喜歡中央的天氣，下雨時在女十四舍前根本就沒辦法撐傘，而且出門必定會淋濕褲子跟鞋子，希望中央的風雨可以少一點。

**Comment by Taoyi-Chiang** [comment]
中央的天氣實在有夠惡劣，根本苦寒之地，我覺得應該可以稱之為臺灣寧古塔。
到底怎麼有辦法又溼風又大？而且那種冷是冷到骨頭的冷，膝蓋根本直接爛掉。

**Comment by poyen0806** [comment]
瑞典人簽，瑞典那裡雖然溫度較低，但是人民比較熱情彌補了這一部分。
中央的天氣真的非常惡劣，冷就算了，還好濕，我都快瘋了。

只好用sklearn等套件預測一下最近的天氣，然後為此先做好準備

**Comment by lichilun** [comment]
我也是很不習慣中央的天氣，跟我長大的地方比起來潮濕太多，天氣冷的時候又超級冷（冰庫），在中央我超容易打噴嚏、過敏，真是受夠中央的天氣....

**Comment by a414414** [comment]
我也很不習慣中壢的天氣，氣候特別惡劣，陰晴不定的。我原本住在中部，一到中央也非常不習慣，尤其中壢這裡會下很奇怪的雨，說是毛毛雨也不像毛毛雨，比較像霧一樣，感覺整個大學都不會適應。
